ocrd_tesserocr/recognize.py

```diff
@@ -33,6 +36,7 @@ def process(self):
                 raise Exception("configured model " + model + " is not installed")
         with PyTessBaseAPI(path=TESSDATA_PREFIX, lang=model) as tessapi:
             log.info("Using model '%s' in %s for recognition at the %s level", model, get_languages()[0], maxlevel)
+            tessapi.SetVariable("glyph_confidences", "2") # populate GetChoiceIterator() with LSTM models, too
```

Comment:
Interesting that tesseract doesn't validate these apparently because tests passed for older versions as well.

Reply:
Yes, I was going to ask Noah or Stefan about them. (As far as I understand, many of them are out-dated anyway. Since Tesseract needs to be downwards-compatible for applications, I guess the variables must be dragged along without further checking.)